api/models/system_phone.py

```diff
@@ -1,8 +1,20 @@
 from api.mixins import TimestampMixin
 from api import db
+from flask_sqlalchemy.query import Query as BaseQuery
+
+
+class SystemPhoneQuery(BaseQuery):
+    def system_phone_exists(self, url_decoded_system_phone):
+        return (
+            self.filter(
+                SystemPhone.phone.contains(url_decoded_system_phone[-10:])
+            ).first()
+            is not None
+        )

```

Comment:
Query to check whether the `system_phone_number` exists or not in the `SystemPhone` table, the query is moved from the `event_service` to the `SystemPhone` model.